static/js/settings_org.js

```diff
@@ -418,7 +418,7 @@ export function populate_auth_methods(auth_methods) {
     if (!meta.loaded) {
         return;
     }
-    const $auth_methods_table = $("#id_realm_authentication_methods").expectOne();
+    const $auth_methods_list = $("#id_realm_authentication_methods").expectOne();
```

Comment:
I think it would be better if these variable name changes are done in a separate commit.

Reply:
My underlying idea here was that those variable/function name changes are directly related to that component change, therefore making a logically whole one. Don't you think they would be more confusing when standing alone? I can change that no problem, just to make sure :)

Reply:
Ok, never mind - to avoid longer discussion here, I've gone ahead and split the commit out (but you're most welcome to comment on my considerations above, I can revert that easily).